File: core/src/androidTest/java/com/emarsys/core/provider/uuid/UUIDProviderTest.java
package com.emarsys.core.provider.uuid;

import static org.junit.Assert.assertNotNull;

import com.emarsys.testUtil.TimeoutUtils;

import org.junit.Rule;
import org.junit.Test;
import org.junit.rules.TestRule;


public class UUIDProviderTest {

    @Rule
    public TestRule timeout = TimeoutUtils.getTimeoutRule();


    @Test
    public void testProvideId_returnsNotNullId() {
        UUIDProvider provider = new UUIDProvider();
        assertNotNull(provider.provideId());
    }

}

File: core/src/androidTest/java/com/emarsys/core/provider/uuid/UUIDProviderTest.kt
package com.emarsys.core.provider.uuid

import io.kotest.matchers.shouldNotBe
import org.junit.Test

class UUIDProviderTest  {
    @Test
    fun testProvideId_returnsNotNullId() {
        val provider = UUIDProvider()
        provider.provideId() shouldNotBe null
    }
}